pkg/operator/ceph/cluster/controller.go

```diff
@@ -133,6 +162,37 @@ func (c *ClusterController) StopWatch() {
 // ************************************************************************************************
 // Add event functions
 // ************************************************************************************************
+func (c *ClusterController) onK8sNodeAdd(obj interface{}) {
+	newNode, ok := obj.(*v1.Node)
+	if !ok {
+		logger.Warningf("Expected NodeList but handler received %#v", obj)
+	}
+	for i := range newNode.Spec.Taints {
+		if newNode.Spec.Taints[i].Effect == "NoSchedule" {
+			logger.Debugf("Skipping -> Node %s is Unschedulable", newNode.Labels[apis.LabelHostname])
```

Comment:
I think a more explanatory message would be something like `"Skipping cluster update. Added node %s is unschedulable."